### DISCUSSION TITLE
Pole figures from alternative directions

### DISCUSSION THREAD

**Discussion by liuqing72** [mtex-toolbox/mtex#1926] [Ask Anything]
Hi friends,

I am working with some texture analysis using pole figures. I can plot pole figures from Z (format) direction. I wonder if there is any command that allows me to plot pole figures from other directions, like X and Y ?

Best,
Liuqing

**Comment by ralfHielscher** [comment]
Hi Liuqing,

with MTEX 6.0.beta1 we have introduced the option to arbitrarily align pole figures. The syntax is as follows
```matlab
how2plot = plottingConvention;
how2plot.outOfScreen = xvector;
how2plot.east = zvector
plotPDF(odf, Miller(1,0,0,odf.CS), how2plot)
```
It would be very nice if you could test this new feature.

All the best,
Ralf.
